commit 74effedbf57c8c8108b34d5d7401b19211c98b18
Merge: bba1f16 48a4410
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Wed Apr 6 16:53:35 2016 -0700

    Merge pull request #3412 from paulusjacobus/RCBugFix
    
    rebased my fork with RCBugFix release in order to push language_nl.h

diff --git a/Marlin/language_nl.h b/Marlin/language_nl.h
index ec1d7dc..889afb3 100644
--- a/Marlin/language_nl.h
+++ b/Marlin/language_nl.h
@@ -38,7 +38,7 @@
 #define WELCOME_MSG                         MACHINE_NAME " gereed."
 #define MSG_SD_INSERTED                     "Kaart ingestoken"
 #define MSG_SD_REMOVED                      "Kaart verwijderd"
-#define MSG_MAIN                            "Main"
+#define MSG_MAIN                            "Hoofdmenu"
 #define MSG_AUTOSTART                       "Autostart"
 #define MSG_DISABLE_STEPPERS                "Motoren uit"
 #define MSG_AUTO_HOME                       "Auto home"
@@ -163,6 +163,11 @@
 #define MSG_HEATING_COMPLETE                "voorverw. kompleet"
 #define MSG_BED_HEATING                     "Bed voorverwarmen"
 #define MSG_BED_DONE                        "Bed is voorverw."
+#define MSG_LEVEL_BED_HOMING                "Homing XYZ"
+#define MSG_LEVEL_BED_WAITING               "Klik voor begin"
+#define MSG_LEVEL_BED_DONE                  "Bed leveling kompleet!"
+#define MSG_LEVEL_BED_CANCEL                "Bed leveling afbreken"
+#define MSG_HOME_OFFSETS_APPLIED            "Home offsets toegepast"
 
 #if ENABLED(DELTA_CALIBRATION_MENU)
   #define MSG_DELTA_CALIBRATE               "Delta Calibratie"
